stack/docker-compose.yml

```
@@ -150,7 +154,7 @@
   backend:
     profiles:
       - backend
-    image: ghcr.io/it-at-m/ehrenamt-justiz/ehrenamt-justiz-backend@sha256:f76593210177e3924b958c451b3981d1d50f04f1d872d0c9d253bde7cc7f19cf
+    image:  ghcr.io/it-at-m/ehrenamt-justiz/ehrenamt-justiz-backend:dev
```

YAMLlint (1.37.1)

[error] too many spaces after colon

(colons)